aidial_adapter_bedrock/server/exceptions.py

```diff
@@ -113,13 +105,7 @@ def to_dial_exception(e: Exception) -> DialException:
     if isinstance(e, DialException):
         return e
 
-    status_code = 500
-    return DialException(
-        status_code=status_code,
-        code=str(status_code),
-        type="internal_server_error",
-        message=str(e),
-    )
+    return InternalServerError(str(e))
```

Comment:
Should we preserve `code=str(status_code)`, as we had before?

Reply:
This is going to be populated automatically in the DIAL HTTPException constructor.